packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/index.tsx

```diff
@@ -191,6 +196,32 @@ export const GitHub = () => {
     );
   }
 
+  if (!originalGit && forkedTemplateSandbox?.git) {
```

Comment:
Great!! Could you also maybe check for `forkedFromSandbox`? So something like:

```
const forkedSandbox = sandbox.forkedTemplateSandbox || sandbox.forkedFromSandbox;
```

If we're ever going to remove the default template from GitHub sandboxes, this will still then work in the future.

Reply:
Done!